src/lib/customTraceType.js

```diff
@@ -25,6 +25,8 @@ export function traceTypeToPlotlyInitFigure(traceType) {
       return {type: 'scatter', mode: 'markers', fill: 'none'};
     case 'area':
       return {type: 'scatter', fill: 'tozeroy'};
+    case 'cartesianArea':
+      return {type: 'scatter', fill: 'tozeroy'};
```

Comment:
why was cartesianArea added?, 
and btw, these:
https://github.com/plotly/react-plotly.js-editor/blob/caf87d269d8bc270054bae0fc99b66bed464f64f/src/lib/customTraceType.js#L8

have to match these:
https://github.com/plotly/react-plotly.js-editor/blob/caf87d269d8bc270054bae0fc99b66bed464f64f/src/lib/customTraceType.js#L26

So area's already in there, we don't need 'cartesianArea'

Reply:
Will remove :) it was held over from when I ported some stuff over from `streambed`